src/replaceHomeCollectionKit.js

```diff
@@ -0,0 +1,230 @@
+import { dashboardNavBarLinks, removeActiveClass } from './navigationBar.js';
+import { renderParticipantHeader } from './participantHeader.js';
+import { findParticipant, renderLookupResultsTable } from './participantLookup.js';
+import fieldMapping from './fieldToConceptIdMapping.js'; 
+import { baseAPI, getIdToken, hideAnimation, showAnimation } from './utils.js';
+
+
+export const renderReplacementKitRequest = (participant) => {
+    const isParent = localStorage.getItem('isParent');
+    document.getElementById('navBarLinks').innerHTML = dashboardNavBarLinks(isParent);
+    removeActiveClass('nav-link', 'active');
+    document.getElementById('replaceHomeCollectionBtn').classList.add('active');
+
+    mainContent.innerHTML = render(participant);
+    bindEventRequestReplacementButton(participant?.['Connect_ID'], participant?.token);
+    bindActionReturnSearchResults();
+};
+
+const renderReplacementScreen = (kitAlreadyReceived) => {
+    return `<div>Request a Home Mouthwash Replacement Kit</div>
+                <div><span class="text-danger">NOTE:</span> Make sure you have verified the mailing address with the participant before entering this request and have updated the address in the User Profile if needed.
+</div>${kitAlreadyReceived ? '<div><span class="text-danger">A home mouthwash kit for this participant has already been received. Please confirm the need for a replacement kit before clicking Request Replacement below.</span></div>' : ''}<div><button id="requestReplacementKitBtn" class="btn btn-primary">Request Replacement</button></div>`;
+}
+
+const render = (participant) => {
+    let template = `<div class="container">`
+    if (!participant) {
+        template +=` 
+            <div id="root">
+                Please select a participant first!
+            </div>
+        </div>
+        `
+    } else {
+        
+        const kitStatus = participant[fieldMapping.collectionDetails]?.[fieldMapping.baseline]?.[fieldMapping.bioKitMouthwash]?.[fieldMapping.kitStatus];
+        const kitStatus1 = participant[fieldMapping.collectionDetails]?.[fieldMapping.baseline]?.[fieldMapping.bioKitMouthwashBL1]?.[fieldMapping.kitStatus];
+
+        let resetTextTemplate = ``;
+        // Is the user's address valid?
+        const poBoxRegex = /^(?:P\.?\s*O\.?\s*(?:Box|B\.?)?|Post\s+Office\s+(?:Box|B\.?)?)\s*(\s*#?\s*\d*)((?:\s+(.+))?$)$/i;
+        const physicalAddressLineOne = participant[fieldMapping.physicalAddress1];
+        const mailingAddressLineOne = participant[fieldMapping.address1];
+        if (
+            (!physicalAddressLineOne || poBoxRegex.test(physicalAddressLineOne)) &&
+            (!mailingAddressLineOne || poBoxRegex.test(mailingAddressLineOne))
+        ) {
+            resetTextTemplate = `<div>Participant address is invalid; cannot send home MW kit.</div>`;
```

Comment:
Not sure about implementation request here. Should we use `mouthwash kit` instead of `MW kit` for clarity?